app/settings.py

```diff
@@ -0,0 +1,131 @@
+"""
+Django settings for app project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = ("django-insecure-y4=ar8*efb!bxot"
+              "#7c_@-3dt16)+77p5s7s2aadrwixv&e0z94")
```

Comment:
The `SECRET_KEY` is hardcoded. For security reasons, it's recommended to load it from an environment variable or a separate configuration file that is not included in version control.